clustered/integration-test/src/test/java/org/ehcache/clustered/management/EhcacheConfigWithManagementTest.java

```diff
@@ -50,9 +50,11 @@
       + "</ohr:offheap-resources>" +
       "</config>\n";
 
-  private static final List<File> MANAGEMENT_PLUGINS = Stream.of(System.getProperty("managementPlugins", "").split(File.pathSeparator))
-    .map(File::new)
-    .collect(Collectors.toList());
+  private static final List<File> MANAGEMENT_PLUGINS = System.getProperty("managementPlugins") == null ?
+    Collections.emptyList() :
+    Stream.of(System.getProperty("managementPlugins").split(File.pathSeparator))
```

Comment:
it seems that in the build.gradle, your removed : 

     systemProperty 'managementPlugins', project.configurations.serverLibs.join(File.pathSeparator)

How is that supposed to return something else than null now ?

More genrally, what is the goal of the new method "copyServerLibs()" compared to what we used to have ?